dll/win32/msv1_0/CMakeLists.txt

```diff
@@ -4,7 +4,10 @@ spec2def(msv1_0.dll msv1_0.spec)
 list(APPEND SOURCE
     lsa.c
     msv1_0.c
+    ntlm/global.c
     precomp.h
```

Comment:
You include it explicitly, so this file can be excluded from CMakeLists

Reply:
precomp.h removed.